tests/openmetrics/test_exposition.py

```diff
@@ -94,6 +95,125 @@ def test_histogram(self):
 # EOF
 """, generate_latest(self.registry))
 
+
```

Comment:
please also add test for exemplars (note that native histograms may have multiple exemplars) see example in https://github.com/prometheus/proposals/blob/main/proposals/2024-01-29_native_histograms_text_format.md